kolibri/utils/tests/test_version.py

```diff
@@ -143,69 +129,45 @@ def test_alpha_0_consistent_version_file(
         """
         Tests that git describe data for an alpha-1 tag generates an a1 version
         string.
         """
-        assert get_version((0, 1, 0, "alpha", 0)) == "0.1.0a1.dev0+git.123.abcdfe12"
+        assert get_version((0, 1, 0)) == "0.1.0a1.dev0+git.123.abcdfe12"
 
     @mock.patch("kolibri.utils.version.get_version_file", return_value=None)
     @mock.patch(
         "kolibri.utils.version.get_git_describe",
-        return_value="v0.1.0-alpha1-123-abcdfe12",
+        return_value="v0.1.0-alpha1",
     )
-    def test_alpha_1_consistent_git(self, describe_mock, file_mock):
+    def test_alpha_1_consistent_git_tag(self, describe_mock, file_mock):
         """
         Tests that git describe data for an alpha-1 tag generates an a1 version
         string.
+        If the version file returns nothing, and get_git_describe returns a version
+        tag like v0.1.0-alpha1 and it matches the major, minor, patch of the version tuple,
+        return the version (without the v) (because that means we are on a tagged commit)
         """
-        assert get_version((0, 1, 0, "alpha", 1)) == "0.1.0a1.dev0+git.123.abcdfe12"
-
-    @mock.patch("kolibri.utils.version.get_version_file", return_value="0.1.0b2")
-    @mock.patch("kolibri.utils.version.get_git_describe", return_value=None)
-    @mock.patch("kolibri.utils.version.get_git_changeset", return_value=None)
-    def test_beta_1_consistent_version_file(
-        self, get_git_changeset_mock, describe_mock, file_mock
-    ):
-        """
-        Test that a VERSION file can overwrite an beta-1 state in case the
-        version was bumped in ``kolibri.VERSION``.
-        """
-        assert get_version((0, 1, 0, "beta", 1)) == "0.1.0b2"
-
-    @mock.patch(
-        "kolibri.utils.version.get_version_file",
-        return_value="0.7.1b1.dev0+git.12.g2a8fe31",
-    )
-    @mock.patch("kolibri.utils.version.get_git_describe", return_value=None)
-    @mock.patch("kolibri.utils.version.get_git_changeset", return_value=None)
-    def test_beta_1_consistent_dev_release_version_file(
-        self, get_git_changeset_mock, describe_mock, file_mock
-    ):
-        """
-        Test that a VERSION file can overwrite an beta-1 state in case the
-        version was bumped in ``kolibri.VERSION``.
-        """
-        assert get_version((0, 7, 1, "alpha", 0)) == "0.7.1b1.dev0+git.12.g2a8fe31"
+        assert get_version((0, 1, 0)) == "0.1.0a1"
 
     @mock.patch("kolibri.utils.version.get_version_file", return_value="0.1.0b1")
     @mock.patch("kolibri.utils.version.get_git_describe", return_value="v0.0.1")
     @mock.patch("kolibri.utils.version.get_git_changeset", return_value="+git123")
-    def test_version_file_ignored(
+    def test_version_file_overrides(
         self, get_git_changeset_mock, describe_mock, file_mock
     ):
         """
-        Test that the VERSION file is NOT used where git data is available
+        Test that the VERSION file is used when git data is available
```

Comment:
Old:
> Test that the VERSION file is NOT used where git data is available

New: 
> Test that the VERSION file is used when git data is available

So, essentially now it is the opposite? Just making sure that is the correct change

Reply:
Yes, previously, we deferred to git data over the VERSION file, now we are doing the opposite (this means we don't ever try to do git things in built environments, but has the effect that in dev environments if you have a VERSION file, then the file will take precedence - but if you're generating a VERSION file you should know what you're doing :) )